Makefile

```diff
@@ -115,8 +115,32 @@ swagger: swagger/docs.go
 
 .PHONY: update-licenses
 update-licenses:
```

Comment:
At first glance I thought that this command hangs and never returns, but is seems to take  a long time to execute and produce output because of the final sort step and lack of parallelism (> 10 min ...lost my patience ). It makes sense to add some echo message that will warn user that it will take a while before the command will execute.

Reply:
Fixed.